R/Learner.R

```diff
@@ -75,7 +75,7 @@
 #' To set hyperparameters, assign a named list to the subslot `values`:
 #' ```
 #' lrn = lrn("classif.rpart")
-#' lrn$param_set$values = list(minsplit = 3, cp = 0.01)
+#' lrn$param_set$set_values(minsplit = 3, cp = 0.01)
```

Comment:
Here we would also have to change the text around this line. But nice catch!

Reply:
check if you're fine with the new text